src/db/models/season.ts

```diff
@@ -0,0 +1,19 @@
+import { randomUUID } from 'crypto';
+import sequelize from 'sequelize';
+import { Column, CreatedAt, Model, Table, Unique, UpdatedAt } from 'sequelize-typescript';
+
+@Table({
+  tableName: 'season',
+})
+export class seasonModel extends Model {
+  @Column({
+    primaryKey: true,
+    allowNull: false,
+    type: sequelize.UUID,
+    defaultValue: sequelize.UUIDV4,
+  })
+  id: typeof randomUUID;
+
+  @Column
+  name: string;
```

Comment:
Can we make the name field nullable: false so that the season name is mandatory to be filled?

Reply:
Yes , we will get that changed

Reply:
done